src/components/CallAssignmentList.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import CallAssignmentList from './CallAssignmentList';
+import { mountWithProviders } from '../utils/testing';
+import {
+    ZetkinCallAssignment,
+    ZetkinMembership,
+} from '../types/zetkin';
+
+describe('CallAssignmentList', () => {
+    let dummyCallAssignments : ZetkinCallAssignment[];
+    let dummyMemberships : ZetkinMembership[];
+
+    beforeEach(()=> {
+        cy.fixture('dummyCallAssignments.json')
+            .then((data : {data: ZetkinCallAssignment[]}) => {
+                dummyCallAssignments = data.data;
+            });
+        cy.fixture('dummyMemberships.json')
+            .then((data : {data: ZetkinMembership[]}) => {
+                dummyMemberships = data.data;
+            });
+    });
```

Comment:
It should not be necessary to reload these fixtures _before every test_. Loading them once, _before all tests_ (using `before()`) should be enough.